ocs_ci/ocs/resources/mcg.py

```diff
@@ -136,7 +137,10 @@ def __init__(self, *args, **kwargs):
             pods = pod.get_pods_having_label(label=constants.RGW_APP_LABEL, namespace=self.namespace)
             assert len(pods) == 0, 'RGW pod should not exist on AWS platform'
 
-        elif config.ENV_DATA.get('platform') in constants.ON_PREM_PLATFORMS:
+        elif (
```

Comment:
@nehaberry @rachael-george @sidhant-agrawal is it always the case that RGW pod is not deployed in independent mode clusters?

Reply:
Yes, the RGW exists in the RHCS cluster and does not run as a pod on the OCS nodes

Reply:
@MeridianExplorer we have discussed this offline, I think we should also add validation for making sure there is no RGW pod running on the OCS node with external clusters (independent mode)